docs/userguide/main.md

```diff
@@ -910,3 +910,73 @@ spec:
     opensearchCredentialsSecret:
       name: dashboards-credentials  # This is the name of your secret that contains the credentials for Dashboards to use
 ```
+
+### Configuring Snapshot Repo (BETA):
+
+This feature is Currently in BETA, you can configure the snapshot repo settings for the OpenSearch cluster through the operator. Using `snapshotRepositories` settings you can configure multiple snapshot repos. Once the snapshot repo is configured a user can create custom `_ism` policies through dashboard to backup the in indexes.
```

Comment:
Please add an explanation what BETA means (not stable, not fully functional, interface could change)

Reply:
Added a `Note` about BETA in my latest commit, please check.